endgame/alphabeta/alphabeta.go

```diff
@@ -366,6 +391,7 @@ func (s *Solver) Solve(plies int) (float32, []*move.Move, error) {
 
 	tstart := time.Now()
 	s.mmCount = 0
+	s.zobrist.Initialize(s.game.Board().Dim(), alphabet.MaxAlphabetSize+1, alphabet.MaxAlphabetSize+1)
```

Comment:
for norwegian, the second argument to this function is smaller than the third argument. This is because Norwegian can make the blank a bunch of letters that are not actually in its tile distribution. I don't think I have this argument readily available but it should be kept in mind that we need to modify it for Norwegian.